modules/3d-tiles/package.json

```diff
@@ -36,14 +36,12 @@
   "dependencies": {
     "@loaders.gl/core": "2.1.0-alpha.3",
     "@loaders.gl/gltf": "2.1.0-alpha.3",
+    "@loaders.gl/tiles": "2.1.0-alpha.3",
     "@loaders.gl/loader-utils": "2.1.0-alpha.3",
     "@loaders.gl/math": "2.1.0-alpha.3",
     "@math.gl/culling": "^3.1.2",
     "@math.gl/geospatial": "^3.1.2",
-    "math.gl": "^3.1.2",
-    "probe.gl": "^3.2.0"
-  },
-  "devDependencies": {
-    "@deck.gl/core": "^8.0.0"
+    "@math.gl/core": "^3.1.3",
```

Comment:
Why dev dependencies?

Reply:
there is no `devDependencies` in `3d-tiles` module